api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p>In return_alerts I don't understand this bit:</p><p> </p><p><code>&nbsp;&nbsp;# Make sure there are more than 3 data points</code></p><p><code>&nbsp;&nbsp;data[3]</code></p><p><code>&nbsp;&nbsp;if(data[-1]["value"]==data[-6]["value"]):</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;return ""</code></p><p> </p><p>I get that <code>data[3]</code> raises an IndexError if that condition's not met, but an <code>assert(len(data) &gt; 3)</code> would be better suited for that. I don't know what the significance of the index -6 is though.</p><p> </p><p> </p><p>Should <code>argentina_string</code> not be empty?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>it will sometimes be empty yes, it depends on the data! </p><p>sure thats a cleaner way.</p>